tests/integration/helpers/factories/db/invenio_records.py

```diff
@@ -95,6 +99,10 @@ def create_from_kwargs(cls, index_name='', **kwargs):
                     object_uuid=instance.record_metadata.id,
                     pid_value=instance.record_metadata.json.get('control_number'),
                     **kwargs).persistent_identifier
+
+        instance.inspire_record = InspireRecord(instance.record_metadata.json,
```

Comment:
Now you can can get the `InspireRecord` model instance directly from the `TestRecordMetadata` factory. @drjova